test/integration/enumerate_collections.test.ts

```diff
@@ -0,0 +1,100 @@
+import { runUnifiedSuite } from '../tools/unified-spec-runner/runner';
+import { TestBuilder, UnifiedTestSuiteBuilder } from '../tools/utils';
+
+const testSuite = new UnifiedTestSuiteBuilder('listCollections with comment option')
+  .initialData({
+    collectionName: 'coll0',
+    databaseName: '',
+    documents: [
+      { _id: 1, x: 11 },
+      { _id: 2, toBeDeleted: true } // This should only be used by the delete test
+    ]
+  })
+  .databaseName('listCollections-with-falsy-values')
+  .test(
+    new TestBuilder('listCollections should not send comment for server versions < 4.4')
+      .runOnRequirement({ maxServerVersion: '4.4.0' })
```

Comment:
technically, the bound is supposed to be inclusive, so this test would, in fact, run and fail on `4.4.0`, it's just that our `4.4` server in the pipeline is the latest patch, which causes this test not to run there; if you look at the drivers spec, you'll see `maxServerVersion` is defined with `x.y.99`, where `y` is the highest minor to which the test would be applicable